api/app/channels/security.py

```
@@ -0,0 +1,459 @@
+            r"\b[A-Z0-9]{8}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{12}\b",
+            re.IGNORECASE,
+        ),
+        PIIType.IBAN: re.compile(r"\b[A-Z]{2}[0-9]{2}[A-Z0-9]{11,30}\b"),
+        PIIType.CREDIT_CARD: re.compile(
+            r"\b(?:4\d{3}|5[1-5]\d{2}|6011|3[47]\d{2})[-\s]?\d{4}[-\s]?\d{4}[-\s]?\d{4}\b"
+        ),
+        PIIType.PHONE_NUMBER: re.compile(
+            r"\b(?:\+?1[-.\s]?)?\(?\d{3}\)?[-.\s]?\d{3}[-.\s]?\d{4}\b"
+        ),
+        PIIType.IP_ADDRESS: re.compile(r"\b(?:\d{1,3}\.){3}\d{1,3}\b"),
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)